pkg/controllers/v1alpha2/mpi_job_controller.go

```diff
@@ -530,15 +531,19 @@ func (c *MPIJobController) syncHandler(key string) error {
 		if err != nil {
 			return err
 		}
-
-		// If the worker is ready, start the launcher.
-		workerReady := worker.Status.ReadyReplicas == workerReplicas
-		if workerReady && launcher == nil {
-			launcher, err = c.kubeClient.BatchV1().Jobs(namespace).Create(c.newLauncher(mpiJob, c.kubectlDeliveryImage))
-			if err != nil {
+	
+		launcher, err = c.kubeClient.BatchV1().Jobs(namespace).Create(c.newLauncher(mpiJob, c.kubectlDeliveryImage))
+		if err != nil {
 				return err
 			}
-		}
+		// If the worker is ready, start the launcher.
```

Comment:
Remove the commented code since it's no longer needed?

Reply:
ok